site/pages/docs/ref/meter-circular/meter-circular-en.hbs

```diff
@@ -0,0 +1,124 @@
+---
+{
+	"title": "Meter circular",
+	"language": "en",
+	"category": "Plugins",
+	"categoryfile": "plugins",
+	"description": "Render a meter into a circular meter styled in CSS.",
+	"altLangPrefix": "meter-circular",
+	"dateModified": "2017-08-03"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>Render a meter into a circular meter styled in CSS.</p>
+
+<h2>Purpose</h2>
+<p>Circular meter are not supported yet by browser and seems to not be explicitly defined in the <a href="https://www.w3.org/TR/html52/rendering.html#the-meter-element-rendering">HTML5.2 spec</a> yet. However, the HTML5 spec provide some indication on how to <a href="https://www.w3.org/TR/html5/rendering.html#the-progress-element-0">render a circular progress ring</a> when the progress element is square.</p>
+
+<h2>Working example</h2>
+<ul>
+	<li><a href="../../../demos/meter-circular/circular-en.html">English examples</a></li>
+	<li><a href="../../../demos/meter-circular/circular-fr.html">French examples</a></li>
+</ul>
+
+
+<h2>How to implement</h2>
+<p>Add the class <code>wb-meter-circular</code> to a meter element</p>
+
+<pre><code>&lt;meter class=&quot;wb-meter-circular&quot; value=&quot;10&quot; max=&quot;100&quot;&gt;10 percent&lt;/meter&gt;</code></pre>
+
+<p>If the meter get updated later on through the jQuery function <code>.val()</code>, this circular meter would be immediately refreshed to represent the latest value. Changing the attribute <code>min, max, low, high, optimum</code> would not initiate a refresh, however it will be taken in consideration when a new value is set to the meter element through jQuery.</p>
+
+
+<h3>Showing the value</h3>
+<p>By default the meter value will be displayed in the center of the circular meter.<p>
+<p>Potential issue: A value number with 4 digit or more may not fully pass the <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html">WCAG success criteria 1.4.4</a> (Level AA).</p>
```

Comment:
We will do some testing in order to determine what are the limits and look at enforcing it in the plugin itself. Just an idea: If the value can not be properly displayed on the circular meter, then it could fallback on the default meter layout.